src/main.py

```diff
@@ -152,7 +198,17 @@ async def help(interaction: Interaction):
 # Ignore non-slash commands
 @client.event
 async def on_message(message: Message):
-    pass
+    if (
+        client.user.mentioned_in(message) or "=chat" in message.clean_content
```

Comment:
I think a prefix like `d.` for "duck" would be more fitting compared to `=`

Reply:
Yeah that's fair.